src/helpers/figures/charts/runtime/chartjs_tooltip.ts

```diff
@@ -117,19 +114,17 @@ export function getPieChartTooltip(
       title: function (tooltipItems) {
         return tooltipItems[0].dataset.label;
       },
+      beforeLabel: (tooltipItem) => tooltipItem.dataset?.label || tooltipItem.label,
```

Comment:
I think it should be reversed
`tooltipItem.dataset?.label || tooltipItem.label` should be `tooltipItem.label || tooltipItem.dataset.label`.

copy-paste went too far :smile: 
![image](https://github.com/user-attachments/assets/8fe40b8e-0f6a-4b3e-a90b-e675891cf935)